src/kernel.rs

```diff
@@ -22,6 +22,12 @@ mod ioctl {
     const VCIO_IOC_MAGIC: u8 = 100;
     const VCIO_IOC_TYPE_MODE: u8 = 0;
 
+    #[cfg(target_pointer_width = "64")]
+    ioctl_readwrite! {
+        /// mailbox_property via ioctl with VCIO_IOC_MAGIC
+        mailbox_property, VCIO_IOC_MAGIC, VCIO_IOC_TYPE_MODE, *mut nix::libc::c_char
+    }
+    #[cfg(not(target_pointer_width = "64"))]
```

Comment:
The original implementation was written assuming a 32‑bit OS, so it should remain guarded by
`#[cfg(target_pointer_width = "32")]` instead of `#[cfg(not(target_pointer_width = "64"))]`.

Reply:
Thank you for your review. I changed.